app/views/branches/_form_conditionals.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= f.fields_for :conditionals do |conditional| %>
+<%= f.fields_for :conditionals, child_index: conditional_index do |conditional| %>
   <div class="govuk-form-group <%= conditional.object.errors[:next].empty? ? '' : 'govuk-form-group--error'  %>" data-index-conditional="<%= conditional.index %>">
```

Comment:
would now be a good time to swap `data-index-conditional` for `data-conditional-index`?

same for the expressions index